manual/root_macros_and_shared_libraries/index.md

```diff
@@ -100,33 +98,21 @@ The example {% include tutorial name="cernstaff" %} calls a ROOT macro to build
    }
 {% endhighlight %}
 
-<p><a name="executing-a-root-macro-from-the-invocation-of-root"></a></p>
-**Executing a ROOT macro from the invocation of ROOT**
 
-You can pass a macro to ROOT in its invocation.
+## Compiling ROOT macros with ACLiC
 
-_**Example**_
+ROOT macros are by default just-in-time compiled with Cling based on the Clang compiler. Alternatively, you can use ACLiC to compile your macro from within a ROOT session to a shard library using the system compiler such as gcc.
 
-The exact kind of quoting depends on the used shell. This example works for bash-like shells.
-
-```
-   $ root -l -b 'myCode.C("some String", 12)'
-```
-
-## Compiling ROOT macros
-
-You can use ACLiC (*Compiling Your Code*) to compile your code and to build a dictionary and a shared library from your ROOT macro. ACliC is implemented in [TSystem::CompileMacro()](https://root.cern/doc/master/classTSystem.html).
-
-When using ACliC, ROOT checks what library really needs to be build and calls your system's C++ compiler, linker and dictionary generator. Then ROOT loads a native shared library.
+ACLiC is implemented in [TSystem::CompileMacro()](https://root.cern/doc/master/classTSystem.html#ac557d8f24d067a9b89d2b8fb261d7e18). When using ACLiC, ROOT checks what library really needs to be build and calls your system's C++ compiler, linker and dictionary generator.
 
 ACLiC executes the following steps:
 
-1. Calls `rootcling` to create automatically a dictionary.
+1. Calling `rootcling` to create automatically a dictionary.
 <br/>For creating a dictionary manually, → see [Using rootcling to generate dictionaries manually](#using-rootcling-to-generate-dictionaries-manually).
 
-2. Calls the the system's C++ compiler to build the shared library.
+2. Calling the system's C++ compiler to build the shared library.
 
-3. If there are errors, it calls the C++ compiler to build a dummy executable to clearly report the unresolved symbols.
+3. Load the shared library and optionally execute the macro.
 
```

Comment:
I would remove this following paragraph. I don't think it is true.

Reply:
So you mean this paragraph:

```
ACLiC adds the classes and functions declared in included files with the same name as the     
ROOT macro files with one of following extensions: `.h`, `.hh`, `.hpp`, `.hxx`,` .hPP`, `.hXX`.
This means that, by default, you cannot combine ROOT macros from different files into one     
library by using `#include` statements; you will need to compile each ROOT macro separately.
```

Right let's remove it, I also think it's too specific to the basic manual.